pkg/resolvers/grbClusterResolver.go

```diff
@@ -1,34 +1,77 @@
 package resolvers
 
 import (
+	"context"
 	"fmt"
+	"time"
 
 	apisv3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
 	"github.com/rancher/webhook/pkg/auth"
 	v3 "github.com/rancher/webhook/pkg/generated/controllers/management.cattle.io/v3"
+	v1 "k8s.io/api/authorization/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apiserver/pkg/authentication/serviceaccount"
 	"k8s.io/apiserver/pkg/authentication/user"
+	authorizationv1 "k8s.io/client-go/kubernetes/typed/authorization/v1"
 )
 
 const (
 	grbSubjectIndex = "management.cattle.io/grb-by-subject"
 	localCluster    = "local"
 )
 
+var (
+	exceptionServiceAccounts = []exceptionSA{
+		{
+			name:      "rancher-backup",
+			namespace: "cattle-resources-system",
+		},
+		{
+			name:      "fleet-agent",
+			namespace: "cattle-fleet-local-system",
+		},
+	}
```

Comment:
If you resolve the names here we can remove the new struct and remove the member variable from the resolver.
```suggestion
	exceptionServiceAccounts = []string{
		serviceaccount.MakeUsername("cattle-resources-system", "rancher-backup"),
		serviceaccount.MakeUsername("cattle-fleet-local-system", "fleet-agent"),
	}
```